docs/content/docs/recipes/signals.md

```diff
@@ -0,0 +1,516 @@
+}
+
+export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
+    /*...*/
+
+    public constructor(
+      public readonly audience: IServiceAudience<IMember>,
+      private readonly signaler: Signaler,
+    ){
+      super();
+
+      this.audience.on("memberRemoved", (clientId: string, member: IMember) => {
+        const clientIdMap = this.posMap.get(member.userId);
+        if (clientIdMap !== undefined) {
+            clientIdMap.delete(clientId);
+
+            //If UserID has no connected clients, remove user from local data
+            if (clientIdMap.size === 0) {
+                this.posMap.delete(member.userId);
+            }
+        }
+        this.emit("mousePositionChanged");
+      });
+
+        /*...*/
+    }
+
+    /*...*/
+}
+```
+We know that `MouseTracker` must be able to share information to all connected clients about where the current client's mouse position is. This is where `Signaler` comes in handy, as it specializes in communicating transient data to connected clients within a Fluid application.
+
+To track the client's mouse position, we use the `mousemouve` event to obtain the client's x and y coordinates. To alert the connected clients of a mouse position change, we then submit a `mouseSignalType` signal with the client's userID and updated position value as the paylod:
+
+```typescript
+export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
+    /*...*/
+
+    public constructor(
+      public readonly audience: IServiceAudience<IMember>,
+      private readonly signaler: Signaler,
+    ){
+      super();
+
+      /*...*/
+
+      window.addEventListener("mousemove", (e) => {
+        const position: IMousePosition = {
+            x: e.clientX,
+            y: e.clientY,
+        };
+        this.sendMouseSignal(position);
+      });
+
+        /*...*/
+    }
+
+    /**
+     * Alert all connected clients that there has been a change to a client's mouse position
+     */
+    private sendMouseSignal(position: IMousePosition) {
+        this.signaler.submitSignal(
+            MouseTracker.mouseSignalType,
+            { userId: this.audience.getMyself()?.userId, pos: position },
+        );
+    }
+
+    /*...*/
+}
+```
+But what is the point of sending a signal if nobody is listening to it? To listen to the `mouseSignalType`, we use `Signaler`'s `onSignal` function. We then update the local data using the payload information from the signal. To display the new presence data, we then emit `mousePositionChanged` to let the view know to re-render:
+
+```typescript
+export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
+    /*...*/
+
+    public constructor(
+        public readonly audience: IServiceAudience<IMember>,
+        private readonly signaler: Signaler,
+    ){
+        super();
+
+        /*...*/
+
+        this.signaler.onSignal(MouseTracker.mouseSignalType, (clientId, local, payload) => {
```

Comment:
Maybe put types on the callback parameters to help folks understand what these are exactly.